submissions/april-1st/submission-april-1st.md

```diff
@@ -0,0 +1,208 @@
+# April 1st Mod Feature
+
+(v 1.2)
+
+
+
+## Working title: Funny Factorio Fan Formulations
+
+
+
+On the special occasion of Friday April 1st, we are proud to present some humorous picks from Factorio's vast modding scene. We will start with mods that only change visuals or audio to add some unique flavors to your eningeering experiences. Afterwards, we look at some crazy new weapons that spice up combat. Finally, we feature a mod with a level of transportation pizzaz that Willy Wonka himself would be proud of.
+
+
+
+### Sights and Sounds
+
+The mod portal features quite a few mods small and large that offer changes to visuals and/or audio. Here is a fun selection:
+
+
+
+**Emotes and Speech Bubbles** 
+
+by [Kiplacon](https://mods.factorio.com/user/Kiplacon)
+
+[This handy mod](https://mods.factorio.com/mod/EmotesAndSpeechBubbles) makes multiplayer interactions a lot livelier, especially when you can't use a mic. Your text in chat also appears over your head. This is already more immersive but the best part is the emotes it provides. The emote menu is opened by clicking the middle mouse button (and customized by right-clicking them). The mod helps enhance your fun and funny moments: There are emotes for partying, coordinating, and of course pressing "F" or declaring "rekt!" when your friend gets hit by a train. 
+
+(Emote gif: easy)
+
+**(Multiplayer image: hard)**
+
+
+
+**Disco Science**
+
+by [danielbrauer](https://mods.factorio.com/user/danielbrauer)
+
+Science is the heart of progress, but [this little mod](https://mods.factorio.com/mod/DiscoScience) also makes it the life of the party! It makes research labs light up with the colors of whichever science packs they are consuming. Excellent for sprucing up your [AFK Zone Discos](https://www.reddit.com/r/factorio/comments/sl4z1v/time_honored_tradition_when_a_player_is_afk_you/)! If you want to go further and add faint music to your disco domes, there is [this other mod](https://mods.factorio.com/mod/LabDansen/metrics).
```

Comment:
Love the reddit reference!

Also, if you could capture the video clip with the LabDansen sounds then it would be so cool! 

BTW, we prefer videos rather than GIFs. It's much smaller. And don't worry if the way you capture it is too big, we can compress it for you :)